WordPress/Classes/ViewRelated/Comments/CommentsViewController.m

```diff
@@ -136,30 +131,20 @@ - (void)configureTableView
     self.tableView.accessibilityIdentifier  = @"Comments Table";
     [WPStyleGuide configureColorsForView:self.view andTableView:self.tableView];
 
-    if ([self usesUnifiedList]) {
-        // Register unified list components
-        UINib *listCellNibInstance = [UINib nibWithNibName:[ListTableViewCell classNameWithoutNamespaces] bundle:[NSBundle mainBundle]];
-        [self.tableView registerNib:listCellNibInstance forCellReuseIdentifier:ListTableViewCell.reuseIdentifier];
+    // Register the cells
+    UINib *listCellNibInstance = [UINib nibWithNibName:[ListTableViewCell classNameWithoutNamespaces] bundle:[NSBundle mainBundle]];
+    [self.tableView registerNib:listCellNibInstance forCellReuseIdentifier:ListTableViewCell.reuseIdentifier];
 
-        UINib *listHeaderNibInstance = [UINib nibWithNibName:[ListTableHeaderView classNameWithoutNamespaces] bundle:[NSBundle mainBundle]];
-        [self.tableView registerNib:listHeaderNibInstance forHeaderFooterViewReuseIdentifier:ListTableHeaderView.reuseIdentifier];
-    } else {
-        // Register the cells
-        NSString *nibName = [CommentsTableViewCell classNameWithoutNamespaces];
-        UINib *nibInstance = [UINib nibWithNibName:nibName bundle:[NSBundle mainBundle]];
-        [self.tableView registerNib:nibInstance forCellReuseIdentifier:CommentsTableViewCell.reuseIdentifier];
-    }
+    UINib *listHeaderNibInstance = [UINib nibWithNibName:[ListTableHeaderView classNameWithoutNamespaces] bundle:[NSBundle mainBundle]];
+    [self.tableView registerNib:listHeaderNibInstance forHeaderFooterViewReuseIdentifier:ListTableHeaderView.reuseIdentifier];
 }
 
 - (void)configureTableViewHeader
 {
-    // Add an extra 10pt space on top of the first header view when displaying comments using the new unified list.
-    // The conditional should be removed when the feature is fully rolled out. See: https://git.io/JBQlU
-    if ([self usesUnifiedList]) {
-        UIView *tableHeaderView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, self.tableView.frame.size.width, 10)];
-        tableHeaderView.backgroundColor = [UIColor systemBackgroundColor];
-        self.tableView.tableHeaderView = tableHeaderView;
-    }
+    // Add an extra 10pt space on top of the first header view when displaying comments using the new unified list. Ref: https://git.io/JBQlU
```

Comment:
Nitpick: referencing the "new unified list" is a bit confusing since it's just the list now. Maybe remove the `when displaying comments using the new unified list` bit?

Reply:
Oh yeah, that makes sense! I've addressed this in https://github.com/wordpress-mobile/WordPress-iOS/pull/17747/commits/b5a820db8e58e4a000cb7e6e43d87655dbb1fe3c. Thank you!